app/recordtransfer/admin.py

```diff
@@ -77,6 +78,12 @@ def format_upload_size(obj: BaseUploadedFile) -> str:
     return get_human_readable_size(int(obj.file_upload.size), 1000, 2)
 
 
+@display(description=gettext("File Link"))
+def media_url(obj: BaseUploadedFile) -> SafeText:
+    """Return the media URL for a BaseUploadedFile instance."""
+    return format_html('<a href="{}">{}</a>', obj.get_file_access_url(), obj.name)
```

Comment:
Similar to my above comment, in case the file doesn't exist, can we do this?

I also think it'd be better if the file opened in a new tab.

```suggestion
    if not obj.file_upload or not obj.exists:
        return mark_safe(gettext("File was removed"))
    return format_html('<a target="_blank" href="{}">{}</a>', obj.get_file_access_url(), obj.name)
```

Reply:
Just tested it, works as expected:
- Inline:
![image](https://github.com/user-attachments/assets/619b949d-8452-4c40-af1d-af2806863bab)
- Admin:
![image](https://github.com/user-attachments/assets/b7f64a47-027e-484a-a164-4cf564c56bdc)